src/dmd/cppmangle.d

```diff
@@ -665,6 +665,8 @@ private final class CppMangleVisitor : Visitor
         //printf("prefix_name(%s)\n", s.toChars());
         if (substitute(s))
             return;
+        if (isStd(s))
+            return buf.writestring("St");
```

Comment:
This looks like it should be part of the last commit.

Reply:
Fixed